### PR TITLE
PAC cluster changes

### PR DESCRIPTION
**Pull request by JermilaPaulDhas**
Created an environment for prod pac cluster
Not setting the env and region as a helm value coz we are using a wildcard host in the helm config

### CODE REVIEW

File: src/com/ft/jenkins/DeploymentUtils.groovy [hunk 1]
```diff
@@ -48,7 +48,7 @@ public executeAppsDeployment(Cluster targetCluster, List<String> appsToDeploy, S
 
       echo "Using app config file ${configurationFileName} to deploy with helm"
 
-      sh "helm upgrade ${app} ${chartFolderLocation} -i -f ${configurationFileName} --set region=${region} --set target_env=${env.name}"
```
Comment (peteclark-ft) [inline src/com/ft/jenkins/DeploymentUtils.groovy]: We may still need this for certain settings - i.e. the ARN for the ssl cert in each region

File: src/com/ft/jenkins/EnvsRegistry.groovy [hunk 2]
```diff
@@ -33,6 +33,16 @@ class EnvsRegistry implements Serializable {
         ("us-" + Cluster.PAC.toString()): "https://pre-prod-us-pac-api.ft.com",
     ]
 
+    Environment prodPac = new Environment()
+    prodPac.name = "prodPac"
```
Comment (peteclark-ft) [inline src/com/ft/jenkins/EnvsRegistry.groovy]: I would switch the naming for pre-prod and prod - i.e. set `prodPac.name = "pac"` and `preProdPac.name = "preProdPac"` ?

File: vars/pacEntryPointForJenkinsfile.groovy [hunk 3]
```diff
@@ -10,7 +10,7 @@ def call() {
   BuildConfig config = new BuildConfig()
   //  todo [sb] adjust these once we have all the envs defined
   config.setPreprodEnvName("pac")
-  config.setProdEnvName("pac")
+  config.setProdEnvName("prodPac")
```
Comment (peteclark-ft) [inline vars/pacEntryPointForJenkinsfile.groovy]: Similarly switch these names here

File: src/com/ft/jenkins/DeploymentUtils.groovy (hunk 1 above)
Comment (sbuliarca) [inline src/com/ft/jenkins/DeploymentUtils.groovy]: Yes, please keep it.

File: src/com/ft/jenkins/EnvsRegistry.groovy (hunk 2 above)
Comment (sbuliarca) [inline src/com/ft/jenkins/EnvsRegistry.groovy]: I think that the strings "prod" and "pre-prod" should appear in the env naming, as they will be used in notifications

File: src/com/ft/jenkins/DeploymentUtils.groovy (hunk 1 above)
Comment (sbuliarca) [inline src/com/ft/jenkins/DeploymentUtils.groovy]: Indeed, please let these values as is.

**Pull request merged**